Makefile

```diff
@@ -129,12 +129,12 @@ pytest-coverage: coverage-erase docs-build pytest-ci
 
 .PHONY: pytest-integration-full
 pytest-integration-full:
-	pytest ${pytest_args} -n auto --live-dbs --etl-settings ${etl_full_yml} test/integration
+	pytest ${pytest_args} -n 4 --no-cov --live-dbs --etl-settings ${etl_full_yml} test/integration
 
 .PHONY: pytest-validate
 pytest-validate:
 	pudl_check_fks
-	pytest ${pytest_args} -n auto --live-dbs test/validate
+	pytest ${pytest_args} -n 4 --no-cov --live-dbs test/validate
```

Comment:
These tests aren't used for generating coverage, and coverage is turned on by default in the coverage configuration. Explicitly disable coverage here to avoid a "failure" after the test runs.

Also, switch to using at most 4 CPUs to avoid running out of memory with our currently very memory inefficient system for running the data validations.